### PR TITLE
Cypress e2e

### PR DESCRIPTION
**Pull request by belkinbelkin**
- Added Cypress 9.2.1
- Added main menu check

### CODE REVIEW

File: cypress/integration/landing.spec.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+describe('Landing Page', () => {
+    beforeEach(() => {
+        cy.visit('/');
+    });
+
+    it('Renders login page', () => {
+        cy.wait(5000);
+        cy.get('.ant-tabs-nav-list').should('be.visible');
+        cy.get('.ant-tabs-tab-btn').first().should('have.text', 'Real Estate');
```
Comment (teamco) [inline cypress/integration/landing.spec.js:11]: use const for cy.get('.ant-tabs-tab-btn')

**Review by teamco — CHANGES_REQUESTED** [review]
fix code - wait(500) and create const for the "similar" selectors

File: cypress/integration/landing.spec.js (hunk 1 above)
Comment (belkinbelkin) [reply cypress/integration/landing.spec.js:10]: nope, doesn't work like that, will handle it separately

**Review by teamco — APPROVED** [review]
ok

**Review by teamco — APPROVED** [review]
ok

**Review by teamco — APPROVED** [review]
ok

**Review by teamco — APPROVED** [review]
ok

**Pull request merged**